com.microsoft.mrtk.spatialmanipulation/PlacementHub.cs

```diff
@@ -0,0 +1,374 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SpatialManipulation
+{
+    public class PlacementHub : MonoBehaviour
```

Comment:
`internal`?

Reply:
Can do!

Reply:
After some additional considerations, I think after this naming hump, I want the "PlacementHub" to be public, as it contains a lot of the smoothing code from the ObjectManipulator, and we let that behavior be extensible in the past.